geom/src/polygon.rs

```diff
@@ -593,50 +601,6 @@ pub struct Triangle {
     pub pt3: Pt2D,
 }
 
-impl Triangle {
-    pub fn new(pt1: Pt2D, pt2: Pt2D, pt3: Pt2D) -> Triangle {
-        Triangle { pt1, pt2, pt3 }
-    }
-
-    fn contains_pt(&self, pt: Pt2D) -> bool {
```

Comment:
I don't remember where I copied this from, but I am quite happy to get rid of it